src/FuelClient.cc

```diff
@@ -186,6 +186,12 @@ class gz::fuel_tools::FuelClientPrivate
   /// license information.
   public: void PopulateLicenses(const ServerConfig &_server);
 
+  /// \brief Checks the provided URI for fuel.ignitionrobotics.org, and
+  /// prints a deprecation warning message if found.
+  /// \param[in] _uri URI to check
+  /// DEPRECATED/DEPRECATION: remove this function in Gazebo H.
```

Comment:
I'm not sure removing this for Gazebo H is a good idea. There might still be worlds/models that use `fuel.ignitionrobotics.org` on Fuel well past Gazebo H.

Reply:
[765bf6c](https://github.com/gazebosim/gz-fuel-tools/pull/294/commits/765bf6c3b3fbb8cb0e996fbe54951883aa9837b4)